kpi/views/v2/asset.py

```diff
@@ -537,7 +537,12 @@ def koboform(self, request, *args, **kwargs):
         return Response({'asset': asset, }, template_name='koboform.html')
 
     def list(self, request, *args, **kwargs):
-        queryset = self.filter_queryset(self.get_queryset())
+        # raising an exception if the default search query without a 
+        # specified field is less than a set length of characters - currently 3
+        try:
+            queryset = self.filter_queryset(self.get_queryset())
+        except BadDefaultSearchException as e:
+            raise exceptions.ValidationError(e)
```

Comment:
As per discussed, let's catch the exception in `SearchFilter`. 
See previous comment.